backend/api/Database/Models/AnalysisResult.cs

```diff
@@ -0,0 +1,42 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Api.Database.Models
+{
+    public class AnalysisResult
+    {
+        [JsonPropertyName("inspectionId")]
+        [Key]
+        public required string InspectionId { get; set; }
+
+        [JsonPropertyName("analysisType")]
+        public required string AnalysisType { get; set; }
+
+        [JsonPropertyName("displayText")]
+        public required string DisplayText { get; set; }
+
+        [JsonPropertyName("value")]
+        public float? Value { get; set; }
+
+        [JsonPropertyName("unit")]
+        public string? Unit { get; set; }
+
+        [JsonPropertyName("class")]
+        public string? Class { get; set; }
+
+        [JsonPropertyName("confidence")]
+        public float? Confidence { get; set; }
+
+        [JsonPropertyName("warning")]
+        public string? Warning { get; set; }
```

Comment:
Is it necessary to have a string. Boolean would be more intuitive to understand and error message could be incorporated into DisplayText